app/src/main/java/com/compscieddy/meetinthemiddle/ChatFragment.java

```diff
@@ -212,10 +227,14 @@ public void updateUI() {
 
   public static class ChatHolder extends RecyclerView.ViewHolder {
 
-    @Bind(R.id.left_arrow) FrameLayout leftArrow;
-    @Bind(R.id.right_arrow) FrameLayout rightArrow;
-    @Bind(R.id.message_container) RelativeLayout messageContainer;
-    @Bind(R.id.message_box) LinearLayout messageBox;
+    @Bind(R.id.left_arrow)
```

Comment:
Put these on the same line please - @DarrenPoh I think you have your auto-indent on again